### PR TITLE
AB#5015

### PR DESCRIPTION
**Pull request by sachin755**
### Proposed changes

*
*

### Related issues

*
*

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by johnjdl** [comment]
@r-joyce please review and merge once the other priority PRs are done and development is bumped. These (5015 and 5017) are lower priorities.

File: opencti-platform/opencti-front/src/private/components/dataEntities/data/responsibleParties/DataSourceResponsiblePartiesDeletion.js
Comment (r-joyce) [inline opencti-platform/opencti-front/src/private/components/dataEntities/data/responsibleParties/DataSourceResponsiblePartiesDeletion.js]: Guessing this should say "Responsible Parties" and not "Risk"?

File: opencti-platform/opencti-front/src/private/components/dataEntities/data/responsibleParties/EntityResponsiblePartyCard.js
Comment (r-joyce) [inline opencti-platform/opencti-front/src/private/components/dataEntities/data/responsibleParties/EntityResponsiblePartyCard.js]: Lorem Ipsum

File: opencti-platform/opencti-front/src/private/components/dataEntities/data/responsibleParties/EntityResponsiblePartyDetails.js
Comment (r-joyce) [inline opencti-platform/opencti-front/src/private/components/dataEntities/data/responsibleParties/EntityResponsiblePartyDetails.js]: Lorem Ipsum

File: opencti-platform/opencti-front/src/private/components/dataEntities/data/responsibleParties/EntityResponsiblePartyDetails.js
Comment (r-joyce) [inline opencti-platform/opencti-front/src/private/components/dataEntities/data/responsibleParties/EntityResponsiblePartyDetails.js]: Hard-coded date

File: opencti-platform/opencti-front/src/private/components/dataEntities/data/responsibleParties/EntityResponsiblePartyDetails.js
Comment (r-joyce) [inline opencti-platform/opencti-front/src/private/components/dataEntities/data/responsibleParties/EntityResponsiblePartyDetails.js]: Lorem Ipsum

File: opencti-platform/opencti-front/src/private/components/dataEntities/data/responsibleParties/EntityResponsiblePartyDetails.js
Comment (r-joyce) [inline opencti-platform/opencti-front/src/private/components/dataEntities/data/responsibleParties/EntityResponsiblePartyDetails.js]: Lorem Ipsum

File: opencti-platform/opencti-front/src/private/components/dataEntities/data/responsibleParties/EntityResponsiblePartyDetails.js
Comment (r-joyce) [inline opencti-platform/opencti-front/src/private/components/dataEntities/data/responsibleParties/EntityResponsiblePartyDetails.js]: Lorem Ipsum

File: opencti-platform/opencti-front/src/private/components/dataEntities/data/responsibleParties/EntityResponsiblePartyDetails.js
Comment (r-joyce) [inline opencti-platform/opencti-front/src/private/components/dataEntities/data/responsibleParties/EntityResponsiblePartyDetails.js]: Hard-coded TLP value

**Review by r-joyce — CHANGES_REQUESTED** [review]
Everything looks good to me besides the occasional Lorem Ipsum or hard-coded value.

@johnjdl @kevinh4 Can we get these fixed before merging?

**Comment by johnjdl** [comment]
@r-joyce looks like Secutree made the fixes you requested. Go ahead and review again and merge when you have a moment.

**Comment by johnjdl** [comment]
@r-joyce Let's hold off on this 5017 and 5019 since it looks like there are some queries provided by Paul that Secutree will need to include.

**Review by r-joyce — APPROVED** [review]
Looks good to me 👍

**Comment by johnjdl** [comment]
@r-joyce Let's hold off on this one.

**Review by r-joyce — APPROVED** [review]
Looks good to me 👍

**Pull request merged**
